lifecycle/test/test_lifecycle.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import unittest
+
+import launch
+import launch.actions
+import launch.event_handlers.on_process_start
+
+
+import launch_ros.actions
+import launch_ros.events
+import launch_ros.events.lifecycle
+
+import launch_testing
+import launch_testing.asserts
+
+import lifecycle_msgs.msg
+
+
+def generate_test_description(ready_fn):
+    talker_node = launch_ros.actions.LifecycleNode(
+        package='lifecycle', node_executable='lifecycle_talker',
+        node_name='lc_talker', output='screen'
+    )
+    listener_node = launch_ros.actions.Node(
+        package='lifecycle', node_executable='lifecycle_listener',
+        node_name='listener', output='screen'
+    )
+    return launch.LaunchDescription([
+        talker_node, listener_node,
+        # When the talker reaches the 'inactive' state, make it take the 'configure' transition.
```

Comment:
Is it really the "inactive" state? Referring to the [design doc](http://design.ros2.org/articles/node_lifecycle.html) I would expect the node to start in the "unconfigured" state.

Reply:
That's just a wrong comment. Revisited all of them in https://github.com/ros2/demos/pull/336/commits/17757acdcc6c41779d87af75673d178de071e12d. PTAL.